contracts/mocks/hts-precompile/HtsPrecompileMock.sol

```diff
@@ -273,6 +262,9 @@ contract HtsPrecompileMock is NoDelegateCall, IHederaTokenService, KeyHelper {
         _setNftTokenInfo(tokenAddress, nftTokenInfo);
     }
 
+    // TODO: implement _post{Action} "internal" functions called inside and at the end of the pre{Action} functions is success == true
```

Comment:
Can you open a ticket and remover the comment in the smart contracts repo to capture this? 
Code to-do's are easy to miss :)